wsmaster/che-core-sql-schema/src/main/resources/che-schema/6.3.0/2__add_workspace_expirations.sql

```diff
@@ -0,0 +1,21 @@
+--
+-- Copyright (c) 2012-2017 Red Hat, Inc.
+-- All rights reserved. This program and the accompanying materials
+-- are made available under the terms of the Eclipse Public License v1.0
+-- which accompanies this distribution, and is available at
+-- http://www.eclipse.org/legal/epl-v10.html
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+CREATE TABLE workspace_expiration (
```

Comment:
consider adding `che` prefix to table name

Reply:
AFAIK we're adding `che_` profix for the tables was initially present in Codenvy. Is there any new agreement about this?

Reply:
Actually we added che prefix to tables that are placed in che, and for to separate che scheme from assemblies that extend che(As far as I remember).